### ISSUE TITLE
linux static build rizin can not debug

### ISSUE THREAD

**Issue by rever1and** [rizinorg/rizin#1133]
ubuntu 20.04 
v0.2.1 static build

### Expected behavior
```
~/Downloads/rizin$ rizin -d ~/pwnkr/loveletter/loveletter
Process with PID 137131 started...
= attach 137131 137131
bin.baddr 0x08048000
Using 0x8048000
asm.bits 32
glibc.fc_offset = 0x00148
 -- Find wide-char strings with the '/w <string>' command
[0xf7f7a120]> 
```

### Actual behavior
```
~/Downloads/rizin$ bin/rizin -d ~/pwnkr/loveletter/loveletter
Process with PID 137847 started...
ptrace_attach: Operation not permitted
Process with PID 137848 started...
ptrace_attach: Operation not permitted
[w] Cannot open 'dbg:///home/reverland/pwnkr/loveletter/loveletter' for writing.
♥ My lover's name is : ♥ My lover's name is : ♥ Whatever happens, I'll protect her...
♥ Whatever happens, I'll protect her...
♥ Impress her upon my memory...
♥ Impress her upon my memory...
♥ Her name echos in my mind...
♥ Her name echos in my mind...
reverland@reverland-ubuntu:~/Downloads/rizin$ I love very much!
I love very much!
```
### Steps to reproduce the behavior
download static build v0.2.1
extract
use that rizin -d ./program

**Comment by XVilka** [comment]
Might be also related to https://github.com/rizinorg/rizin/issues/982

**Comment by ret2libc** [comment]
Please provide all the info like operating system, architecture, etc. Also the tested binary would be great to have!

**Comment by ret2libc** [comment]
This is quite an important bug to fix. I've put it in 0.4.0, but it would actually be great to fix it even before 0.3.0.
